scripts/__tests__/build-blog-data-integration.test.js

```
@@ -0,0 +1,432 @@
+/**
+ * Integration tests for the build-blog-data.js script
+ *
+ * This test suite verifies that the complete blog data build process
+ * properly handles sanitization, file operations, and data generation.
+ */
+
+const fs = require('node:fs')
+const path = require('node:path')
+const { execSync } = require('child_process')
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)